### ISSUE TITLE
Does OpenLDAP is supported????

### ISSUE THREAD

**Issue by alepensato** [philbertphotos/osticket-multildap-auth-plugin#75]
Does OpenLDAP is supported???? If no, please change the name to avoid user frustration

**Comment by philbertphotos** [comment]
Yes and no ... I stripped OpenLDAP some years ago to retool it and forgot to put it back since most people seem to favor AD. I will start updating the code to support OpenLdap again sorry for that.

**Comment by alepensato** [comment]
Ok I have trouble with defaut ldap plugin that not allow clients to login in but only agent, with the same crededential, and i do not want change the system, but now is a big problem

**Comment by teachmestuff-az** [comment]
On the same topic, I'm attempting to integrate osTicket to work with the internal LDAP server from a Zimbra mail server. Is it possible to accomplish this with the current code? The setup is on Ubuntu.

**Comment by philbertphotos** [comment]
> On the same topic, I'm attempting to integrate osTicket to work with the internal LDAP server from a Zimbra mail server. Is it possible to accomplish this with the current code? The setup is on Ubuntu.

Yep it can quite easy ... just putting back OpenLdap support.

**Comment by alepensato** [comment]
many thanks, i hope that this will do the work and solve the problem with default ldap plugin that does not works

**Comment by alepensato** [comment]
@philbertphotos 
any updates?
